src/components/Post/Details.tsx

```diff
@@ -19,6 +21,18 @@ export const postDetailsTestIds = {
 };
 
 const PostDetails = ({ post }: PostDetailsProps) => {
+  const renderCommaSeparatedTags = () => {
+    const tags = post.tags.map((tag) => (
+      <Link key={tag} href={`/tags/${tag}`}>
+        <a className="hover:underline" href={`/tags/${tag}`}>
```

Comment:
@tyrro Is this will work with Hoang's PR?
Or should we build it with the current base URL?

Reply:
I think it'll work as Hoang didn't seem to change any `href` in this PR too, regardless I'll test his branch if this one gets merged before!